### ISSUE TITLE
Non-free symbol doesn't show

### ISSUE THREAD

**Issue by GinoGiotto** [tirix/metamath-web#16]
Mentioned in https://github.com/tirix/metamath-web/issues/10

>> http://metamath.tirix.org/mpeuni/bm1.1 The hypothesis bm1.1.1 doesn't show. Is it because the non-freeness symbol is missing?
>
> I tried and it worked for me. Maybe you need to refresh the page? (or clear your cache?)

It works on PC, but on Android it appears like this:

![Screenshot_20231030_090118_Chrome](https://github.com/tirix/metamath-web/assets/73717712/ba80aecd-e1db-469b-a6a5-771846447441)

I cleared my cache and doesn't work. It seems to happen whenever there is a non-free symbol in the statement or proof step. Instead of the appropriate symbols a small `no glyph` label appears as shown in the picture.

Edit: On Android it doesn't work with Google, Bing and DuckDuckGo, but it does work with Firefox.

**Comment by tirix** [comment]
"No glyph" suggests that the mobile browsers do not manage to fallback to a font where the glyph exists.
I imagine this might be fixed by loading and applying the XITS font family:
```
  @font-face {
    font-family: XITSMath-Regular;
    src: url(xits-math.woff);
  }
  .math { font-family: XITSMath-Regular }
``` 

(On a side you, you should not get the "could not format x" line for `vx`, but that's a different story)

**Comment by GinoGiotto** [comment]
> "No glyph" suggests that the mobile browsers do not manage to fallback to a font where the glyph exists.

I opened the page with another Android mobile phone just to double check, and the same problem appears (I'm sure metamath was never opened there before). On a Windows machine the issue does not show for the browsers I have (Chrome, Edge, DuckDuckGo), so for me it's hard to test if your solution works. I propose to implement it and see if it solves the problem?

> (On a side note, you should not get the "could not format x" line for vx, but that's a different story)

Good catch, I haven't even noticed, it shows vx correctly now.

**Comment by tirix** [comment]
I've merged #25 which includes the changes mentioned here, which automatically closed this issue, but actually I'm not sure this is really fixed. 
I'm reopening until you confirm everything is now fine.

**Comment by GinoGiotto** [comment]
It's working now! I tested on Android mobile with the browers that weren't working before (Chrome, Bing and DuckDuckGo) and now the non-freenes symbol is showing correctly on all of them (btw for verification I simply opened a new page in incognito mode, and it worked even without clearing the cache).
